.ci/common/test.sh

```diff
@@ -83,8 +83,9 @@ install_nvim() {
     exit 1
   }
 
+  local genvimsynf=syntax/vim/generated.vim
   # Check that all runtime files were installed
-  for file in doc/tags syntax/vim/generated.vim $(
+  for file in doc/tags $genvimsynf $(
     cd runtime ; git ls-files | grep -e '.vim$' -e '.ps$' -e '.dict$' -e '.py$' -e '.tutor$'
```

Comment:
It seems like `doc/tags $genvimsynf ` should be `runtime/doc/tags "runtime/$genvimsynf"`